app/models/editor_post.rb

```diff
@@ -0,0 +1,6 @@
+class EditorPost < ActiveRecord::Base
+  belongs_to :editor, class_name: User
+  belongs_to :post
+
+  validates_presence_of :editor, :post
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.